### PR TITLE
Upgrade liquibase (WOR-441).

### PR DESCRIPTION
**Pull request by cahrens**
Addressing security vulnerability, no breaking changes noted between 4.2.1 and 4.8.0. https://github.com/liquibase/liquibase/releases

https://broadworkbench.atlassian.net/browse/WOR-441

### CODE REVIEW

File: integration/gradle.lockfile [hunk 1]
```diff
@@ -203,7 +203,7 @@ org.junit:junit-bom:5.7.0=compileClasspath,runtimeClasspath,testCompileClasspath
 org.junit:junit-bom:5.8.2=spotbugs
 org.jvnet.mimepull:mimepull:1.9.11=compileClasspath,testCompileClasspath
 org.jvnet.mimepull:mimepull:1.9.13=runtimeClasspath,testRuntimeClasspath
-org.liquibase:liquibase-core:4.2.1=runtimeClasspath,testRuntimeClasspath
+org.liquibase:liquibase-core:4.8.0=runtimeClasspath,testRuntimeClasspath
```
Comment (msilva-broad) [inline integration/gradle.lockfile:206]: this is a decent version bump... I've seen breaking changes sneak in with this sort of jump... Is anything dramatic listed on liquibase site/release-docs?

Comment (cahrens) [reply integration/gradle.lockfile:206]: I read through everything and provided links to the major release in the ticket comment-- https://broadworkbench.atlassian.net/browse/WOR-441?focusedCommentId=64825

There were no mention of breaking changes in any of the releases. Note that liquibase is used for database migrations-- I am hoping that all the existing migrations are applied in some part of the test or build process, but I don't actually know that.

Comment (blakery) [reply integration/gradle.lockfile:206]: I double checked, and I didn't see anything that looked like a breaking change either, unless we were referencing externally defined entities with the xml parser.

**Review by blakery — APPROVED** [review]
LGTM

**Review by marctalbott — APPROVED** [review]

**Pull request merged**